app/helpers/title_helper.rb

```diff
@@ -0,0 +1,20 @@
+module TitleHelper
+  def format_label(count, item)
+    count > 1 ? item.pluralize : item
+  end
+
+  def format_title(searched, page_title, current_user, item_label, count, organisation_name)
+    if searched.present?
+      actual_title = support_user_sab_nav?(current_user, organisation_name) ? organisation_name : page_title
+      "#{actual_title} (#{count} #{item_label} matching ‘#{searched}’)"
+    else
+      support_user_sab_nav?(current_user, organisation_name) ? "#{organisation_name} (#{page_title})" : page_title
+    end
+  end
+
+private
+
+  def support_user_sab_nav?(current_user, organisation_name)
```

Comment:
Should this be `sub_nav`?

Reply:
Or maybe `support_sub_nav`?

Ultimately, we’d want likely want this title behaviour – i.e. showing the selected sub navigation label in brackets – whenever there is sub navigation on a page (some upcoming views for schemes will need this), but for now this works perfectly.